### ISSUE TITLE
Back navigation breaks the auth flow

### ISSUE THREAD

**Issue by SinnySupernova** [kanidm/kanidm#3218]
### I did this

when logging in using browser:
1. click begin
2. click TOTP and password
3. think "fumble, go back"
4. click back button for browser navigation
5. click Passkey

### I expected the following
auth flow works and I can still log in using passkeys

### This happened instead
`
Error
An unrecoverable error occured. Please contact your administrator with the details below.
Error Code: InvalidAuthState("session already finalised!")
`
There's no decent way to recover and this is *most definitely* bad UX

### Kanidm version details

- Output of `kanidm(d) version`: 
- Are you running it in a container? If so, which image/tag?: docker image v1.4.2
- If not a container, how'd you install it: --
- Operating System / Version (On Unix please post the output of `uname -a`): --

### Any other comments
I'm not familiar with the state of htmx webui so this might be already fixed somewhere :sweat_smile:

**Comment by yaleman** [comment]
Yep bad UX but likely requires the auth state machine to have some fun spammers thrown at it. Thanks for calling this out.
